taskcluster/kinds/evaluate-quantized/kind.yml

```diff
@@ -111,7 +111,6 @@ tasks:
                     --dataset_prefix    "$MOZ_FETCHES_DIR/{dataset_sanitized}"
                     --vocab_src          "$MOZ_FETCHES_DIR/vocab.{src_locale}.spm"
                     --vocab_trg          "$MOZ_FETCHES_DIR/vocab.{trg_locale}.spm"
-                    --shortlist         "$MOZ_FETCHES_DIR/lex.s2t.pruned"
```

Comment:
I think we should not disable shortlisting in the pipeline evals. It's a bug in the Firefox integration that should be fixed. We can just assume that the prod results are slightly better. Also, when properly implemented, we saw that it almost doesn't affect quality. 

Unless we make a big decision to disable shortlisting permanently. Then it should be made optional and disabled everywhere, including in the models repo evals.